src/main/java/com/incognia/IncogniaAPI.java

```diff
@@ -111,6 +112,23 @@ public TransactionAssessment registerPayment(
         "api/v2/authentication/transactions", requestBody, TransactionAssessment.class);
   }
 
+  public void registerFeedback(
```

Comment:
i'm missing the possibility to verify if the operation was sucessfull (Something like check the response code at least)

Reply:
if no exceptions are thrown, it was successful (this is the same pattern as the other methods)

Reply:
yeah i guess you're right